Adds react native sign out guide (#1936)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/sign-users-out/define-signout-callback/react-native/addbaseuri.md

> Signing out of Okta requires the app to open a browser and navigate to the [end session endpoint](https://developer.okta.com/docs/reference/api/oidc/#logout). Okta destroys the user's session and immediately redirects back to your application. To do this, you must define a callback route for the sign-out process.

Reviewer: I see you added a note about this being browser based flow only in another file. Do we want the same here?

> Make sure that the `endSessionRedirectUri` configured at `createConfig` is the same as the one defined in the **Logout redirect URI** section. For example if you are initializing your client via a config file:

Reviewer: Might be worth adding a little more context here: "**Logout redirect URI** section of the okta admin dashboard" or similar.

If you'd like to use a custom sign-out flow, you can skip to the next step: <GuideLink link="../sign-out-of-okta">Sign users out of Okta</GuideLink>.

If you'd like to use the browser sign-out flow, you will need to define a callback route.

> Make sure that the `endSessionRedirectUri` configured at `createConfig` is the same as the one defined in the **Logout redirect URI** section of the okta admin dashboard. For example if you are initializing your client via a config file:

Reviewer 2: ```suggestion
Make sure that the `endSessionRedirectUri` configured at `createConfig` is the same as the one defined in the **Logout redirect URI** section of the Okta Admin Console. For example if you are initializing your client with a config file:
```
